DESCRIPTION

```diff
@@ -3,30 +3,29 @@ Title: PHS R Templates
 Authors@R: c(
     person("Alan", "Yeung", email = "alan.yeung@phs.scot", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5226-3695")),
```

Comment:
Adding PHS as the copyright holder - I've but the Data Science email but feel free to use whatever you think (or possibly none).
```suggestion
    person("Public Health Scotland", , , "phs.datascience@phs.scot", role = "cph"),
    person("Alan", "Yeung", email = "alan.yeung@phs.scot", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5226-3695")),
```